src/pages/testing/upload.js

```diff
@@ -1,24 +1,36 @@
+/* eslint-disable react-hooks/exhaustive-deps */
 import React, { useState } from "react";
 
 const Uploadform = () => {
     const [destination, setDestination] = useState("");
-
-    const handleDestinationChange = (event) => {
-        setDestination(event.target.value);
+    const [files, setFiles] = useState(null);
+    const [uploadData, setUploadData] = useState({});
+
+    const handleSubmit = async (event) => {
+        const formData = new FormData();
+        formData.append("file", files[0]);
+
+        const uploadData = await fetch(`/api/upload?dest=${destination}`, {
+            method: "POST",
+            credentials: "include",
+            body: formData,
+        })
+            .then((resp) => resp.json())
+            .catch((e) => console.log(`error ${e.message}`));
+
+        setUploadData(uploadData);
```

Comment:
I did see a few github issues and whatnot that said to avoid using hooks inside `await`, but I'm guessing this PR is just for testing the upload functionality?

Reply:
Ah, I missed that. I'll fix it.

Reply:
Dug into this more, the caution was specific to `useEffect`, so never mind! 😊